server/src/main/java/org/apache/calcite/server/ServerDdlExecutor.java

```diff
@@ -559,6 +561,53 @@ public void execute(SqlCreateTable create,
     }
   }
 
+  /** Executes a {@code CREATE TABLE LIKE} command. */
+  public void execute(SqlCreateTableLike create,
+      CalcitePrepare.Context context) {
+    final Pair<CalciteSchema, String> pair = schema(context, true, create.name);
+    final JavaTypeFactory typeFactory = context.getTypeFactory();
```

Comment:
`typeFactory` and `rowType` can be moved to last because them are used by last code.